deploy/us-east-2-dev/services.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: storetheindex-0-storetheindex
+  namespace: default
+  labels:
+    app: storetheindex
+    release: storetheindex-0
+spec:
+  type: ClusterIP
+  selector:
+    app: storetheindex
+    release: storetheindex-0
+  ports:
+    - name: finder
+      port: 3000
+      protocol: TCP
+      targetPort: 3000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: storetheindex-0-storetheindex-0
+  namespace: default
+  labels:
+    app: storetheindex
+    release: storetheindex-0
+spec:
+  ports:
+  - name: admin
+    port: 3002
+    protocol: TCP
+    targetPort: 3002
+  selector:
+    app: storetheindex
+    release: storetheindex-0
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: storetheindex-0-storetheindex-1
```

Comment:
Can we give services here a more descriptive name, say `finder`? Ditto re other services (and objects in storetheindex namespace).

P.S. I am not sure why pretty much ever object name is qualified by `storetheindex`? I imagine the separation of namespace is enough, no? 

My vote: Simpler, more descriptive names would make working with kubectl and debugging easier.